examples/dragon.rs

```diff
@@ -0,0 +1,41 @@
+// Draw a dragon curve, more specifically a highway dragon.
+// (https://en.wikipedia.org/wiki/Dragon_curve)
+
+extern crate turtle;
+
+use turtle::Turtle;
+use turtle::Color;
+
+fn main() {
+    let mut turtle = Turtle::new();
+
+    turtle.set_background_color("#112244");
+
+    turtle.pen_up();
+    turtle.backward(160.0);
+    turtle.left(90.);
+    turtle.forward(110.);
+    turtle.pen_down();
+    turtle.set_speed(10);
+
+    dragon(&mut turtle, 90., 11, 0., 255.);
+
+    turtle.hide();
+}
+
+fn dragon(turtle: &mut Turtle, turn: f64, depth: usize, c0: f64, c1: f64) {
```

Comment:
Could you add some documentation to this function regarding the meaning of each of these parameters? If you can't also give them more detailed names, it would be really helpful to someone playing with this example in the future if you could write down some information about what each variable and parameter means and what it does to the output.